nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -134,9 +138,22 @@ protected void notifyEvent(CoreSession session, String eventType, DocumentModel
     }
 
     protected NuxeoPrincipal getAuthor(DocumentModel docModel) {
-        String[] contributors = (String[]) docModel.getProperty("dublincore", "contributors");
-        UserManager userManager = Framework.getService(UserManager.class);
-        return userManager.getPrincipal(contributors[0]);
+        String author = null;
+        if (docModel.hasSchema(COMMENT_SCHEMA)) {
+            // means annotation / comment
+            author = (String) docModel.getPropertyValue(COMMENT_AUTHOR);
+        }
+        if (StringUtils.isBlank(author)) {
+            String[] contributors = (String[]) docModel.getPropertyValue("dc:contributors");
+            author = contributors[0];
+        }
+
+        NuxeoPrincipal principal = Framework.getService(UserManager.class).getPrincipal(author);
+        // If principal doesn't exist anymore
+        if (principal == null) {
+            log.debug("Principal not found: " + principal);
```

Comment:
Ok author instead of principal but this is the old Log interface so it doesn't have the args substitution method.
 here I need to `log.debug("Principal not found: " + author);`
But I'll do this on master